server/sonar-server/src/main/java/org/sonar/server/es/EsClientProvider.java

```diff
@@ -65,19 +66,19 @@ public EsClient provide(Configuration config) {
       Settings.Builder esSettings = Settings.builder();
 
       // mandatory property defined by bootstrap process
-      esSettings.put("cluster.name", config.get(CLUSTER_NAME).get());
+      esSettings.put("cluster.name", config.get(CLUSTER_NAME.getKey()).get());
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjl%2FSONAR-10300%2Fforbid_system_properties_in_settings_ws&issues=AWEnWsXDgvt_XO9IEnRl&open=AWEnWsXDgvt_XO9IEnRl)

[ISSUE_KEY]: AWEnWsXDgvt_XO9IEnRl